### PR TITLE
Feat/search

### PR DESCRIPTION
**Pull request by louismagdaleno**

### CODE REVIEW

File: socialtours/src/components/Search.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+			return (
+				<Section>
+                    <SearchBar />
+                    <SearchResults placeholder="Search for an influencer"
+                                name="search"
+                                value={this.state.search}
+                                onChange={this.handleChange} 
+                    />
+                    <SearchBtn onClick={this.fetchResults}>Search</SearchBtn>
+                </Section>
```
Comment (jpwigdor) [inline socialtours/src/components/Search.js]: When typing a test influencer name into the search bar, it appears it's not editing the state. Perhaps the methods should be on the `<SearchBar />` styled div? currently it looks like it's an empty input field.

> search/Search%20for%20an%20influencer 404 (Not Found)

**Comment by wmemorgan** [comment]
[Be able to search for influencers I like](https://trello.com/c/ewmKUeVk/92-be-able-to-search-for-influencers-i-like)

File: socialtours/src/components/Search.js (hunk 1 above)
Comment (louismagdaleno) [reply socialtours/src/components/Search.js]: great catch! i'm super embarassed that i missed that.  i have made the correction and pushed up the change. the state is now mutating correctly.

**Comment by jpwigdor** [comment]
WIth the changes, once the Back-End endpoint is created this should work.

File: socialtours/src/components/Search.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	handleChange = (e) => {
+		e.preventDefault();
+		this.setState({
+			search: e.target.value
+		})
+	}
+
+	fetchResults = e => {
+		e.preventDefault();
+		axios.get(API_ENDPOINT + `/search/${this.state.search}`)
```
Comment (bcuz) [inline socialtours/src/components/Search.js]: think this line should be `axios.get(API_ENDPOINT + `**/api**/search/${this.state.search}`)
`

**Comment by bcuz** [comment]
@louismagdaleno Search.js doesnt have our prettier formatting applied to it

File: socialtours/src/components/Search.js (hunk 2 above)
Comment (louismagdaleno) [reply socialtours/src/components/Search.js]: fixed

**Pull request merged**